next.config.mjs

```diff
@@ -7,6 +7,17 @@ const nextConfig = {
       allowedOrigins: ["app.localhost:3000"],
     },
   },
+
+  experimental: {
+    turbo: {
+      rules: {
+        "*.svg": {
+          loaders: ["@svgr/webpack"],
+          as: "*.js",
+        },
+      },
+    },
+  },
```

Comment:
remove this, turbo will be setup natively in next 15 I thin we won't need that experimental turbo

Reply:
good